terraform/modules/check-user-expiry-lambda/s3_objects/email_from.txt.j2

```diff
@@ -0,0 +1 @@
+DataWorks Access Management <{{email_address_dataworks_access_management}}>
```

Comment:
this needs to be sourced from `aws-common-infrastructure` remote state

Reply:
This is done, thanks.